indexers/archive/src/indexer/extract/index.ts

```diff
@@ -19,4 +19,10 @@ switch (process.env.EXTRACTION_SOURCE) {
   }
 }
 
-export default extractBlock;
+export default async function extract(startBlock: number, endBlock: number) {
```

Comment:
Nitpicking: 
No need to call it extract, we can just do:
```suggestion
export default async (startBlock: number, endBlock: number) => {
```